app/Http/routes.php

```diff
@@ -45,6 +45,8 @@
     Route::get('talks/{id}/delete', 'TalksController@destroy');
     Route::get('conferences/{id}/delete', 'ConferencesController@destroy');
 
+    Route::get('talks/{talkId}/revisions/{revisionId}', 'TalkRevisionsController@show');
```

Comment:
Should just use `talk-revisions/{id}` or similar, no need to nest, good read: http://weblog.jamisbuck.org/2007/2/5/nesting-resources

_Or_, like you suggested in Slack, `talks/{id}?revision=2r89ce298`, which I think I actually like the looks of better.